runtime/kusama/src/lib.rs

```diff
@@ -1353,13 +1326,7 @@ construct_runtime! {
 		AuthorityDiscovery: pallet_authority_discovery::{Pallet, Config} = 12,
 
 		// Governance stuff.
-		Democracy: pallet_democracy::{Pallet, Call, Storage, Config<T>, Event<T>} = 13,
```

Comment:
Will make a follow-up issue to remove the data associated with all of these pallets. Or you want to tackle it here?

Reply:
Follow-up, I think. It need not hold things up.

Reply:
Tracked here: https://github.com/paritytech/polkadot/issues/6749